.github/workflows/check-for-workflow-run.js

```diff
@@ -1,19 +1,19 @@
 // Check for a successful previous run of a workflow.
 // This could make a decent reusable action itself.
-module.exports = async ({github, context, core}, workflowID, branch, sha = null) => {
+module.exports = async ({github, context, core}, {workflow_id, sha, ...config}) => {
```

Comment:
```suggestion
module.exports = async ({github, context, core, workflow_id, sha, ...config}) => {
```

Might as well pass everything in one object parameter rather than two, right?

Reply:
Good point!  Amended existing commit.